### PR TITLE
Set MIN_PER_PARTITION_MEMORY to 10 MiB instead of 4 MiB to match the change in the backend limit.

### PR DESCRIPTION
**Pull request by dpcollins-google**
------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by dpcollins-google** [comment]
@scwhittle

File: sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsublite/internal/SubscribeTransform.java [hunk 1]
```diff
@@ -56,7 +56,7 @@ public class SubscribeTransform extends PTransform<PBegin, PCollection<Sequenced
 
   private static final long MEBIBYTE = 1L << 20;
   private static final long SOFT_MEMORY_LIMIT = 512 * MEBIBYTE;
-  private static final long MIN_PER_PARTITION_MEMORY = 4 * MEBIBYTE;
+  private static final long MIN_PER_PARTITION_MEMORY = 10 * MEBIBYTE;
```
Comment (scwhittle) [inline sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsublite/internal/SubscribeTransform.java:59]: LGTM but would this be better as a pipeline option or something that could be configured on PubsubLiteIO?  There may perhaps be cases where the user wants to specify higher per-partition min/max or total memory limit.

Not yet a committer though so someone else will have to merge

**Review by scwhittle — APPROVED** [review]

File: sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsublite/internal/SubscribeTransform.java (hunk 1 above)
Comment (dpcollins-google) [reply sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsublite/internal/SubscribeTransform.java:59]: No, it should really match the service max message size. I don't see this changing again in the future though

**Comment by liferoad** [comment]
Run Java_GCP_IO_Direct PreCommit

**Review by bvolpato — APPROVED** [review]

**Pull request merged**